lib/go-tc/deliveryservices.go

```diff
@@ -390,6 +392,23 @@ func (ds *DeliveryServiceNullable) validateTypeFields(tx *sql.Tx) error {
 		"orgServerFqdn": validation.Validate(ds.OrgServerFQDN,
 			validation.By(requiredIfMatchesTypeName([]string{DNSRegexType, HTTPRegexType}, typeName)),
 			validation.NewStringRule(validateOrgServerFQDN, "must start with http:// or https:// and be followed by a valid hostname with an optional port (no trailing slash)")),
+		"slicePluginBlockSize": validation.Validate(ds,
+			validation.By(func(dsi interface{}) error {
+				ds := dsi.(*DeliveryServiceNullable)
+				if ds.RangeRequestHandling != nil {
+					if *ds.RangeRequestHandling == 3 {
+						return validation.Validate(ds.SlicePluginBlockSize, validation.Required,
+							// Per Slice Plugin implementation
+							validation.Min(262144),   // 256KB
+							validation.Max(33554432), // 32MB
```

Comment:
nit but these are actually 256KiB and 32MiB not 256KB and 32MB, respectively.

Reply:
:)